TouchEffect.UWP/PlatformTouchEff.cs

```diff
@@ -24,11 +30,41 @@ public static void Preserve()
         private bool _pressed;
         private bool _intentionalCaptureLoss;
 
+        private Storyboard _pointerDownStoryboard;
+        private Storyboard _pointerUpStoryboard;
+
         protected override void OnAttached()
         {
             _effect = Element.GetTouchEff();
             _effect.Control = Element as VisualElement;
             _effect.ForceUpdateState(false);
+            if (_effect.NativeAnimation && _effect.UWPTilt)
+            {
+                Debug.WriteLine("PLAY NATIVE ANIMATION!!");
+                var nativeControl = Container;
+                if (String.IsNullOrEmpty(nativeControl.Name))
+                    nativeControl.Name = Guid.NewGuid().ToString();
+                if (nativeControl.Resources.ContainsKey("PointerDownAnimation"))
+                    _pointerDownStoryboard = (Storyboard)nativeControl.Resources["PointerDownAnimation"];
+                else
+                {
+                    _pointerDownStoryboard = new Storyboard();
+                    var downThemeAnimation = new PointerDownThemeAnimation();
+                    Storyboard.SetTargetName(downThemeAnimation, nativeControl.Name);
+                    _pointerDownStoryboard.Children.Add(downThemeAnimation);
+                    nativeControl.Resources.Add(new KeyValuePair<object, object>("PointerDownAnimation", _pointerDownStoryboard));
+                }
+                if (nativeControl.Resources.ContainsKey("PointerUpAnimation"))
```

Comment:
i prefer guard pattern

if (nativeControl.Resources.ContainsKey("PointerUpAnimation"))
{
// do action
retrun
}

then without "else"

Reply:
How would that work? Because returning then will prevent the next `if else` clause from ever running

Reply:
Okay, then please use parenthesis everywhere.